Vagrantfile.example

```diff
@@ -72,7 +74,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         # issue and our use case, the default cpu_mode is overridden.
         domain.cpu_mode = "host-passthrough"
         domain.graphics_type = "spice"
-        domain.memory = 3076
+        domain.memory = 3096
```

Comment:
3076 is the correct value for 3GB exactly. It is 3*1024.

Reply:
I'm pretty sure my motivation was never that advanced :)  I think the centos-fips branch where we first made these patches was from when we provisioned with 2gb and I increased it to a number that sounded right without actually doing the match lol.